README.md

```diff
@@ -63,6 +63,13 @@ brew install clang-format lcov doxygen
 
 You will need to install CMake and Visual Studio, including the Visual C++ compiler. For building on Windows, please follow the [Windows build instructions](dev-docs/build_windows.md).
 
+* FreeBSD
```

Comment:
I think we're keeping this alphabetical - maybe place this between Debian and macOS?

Reply:
Oops, sorry, got trigger-happy.